pnpm-workspace.yaml

```diff
@@ -1,7 +1,6 @@
 packages:
   - "android"
   - "apps/*"
-  - "!apps/envio"
```

Comment:
I don't think this is needed

Reply:
Thank you! It should automatically detect given the child's `pnpm-workspace.yaml` but will leave it there in case 🙇‍♂️